source/gameengine/Ketsji/KX_NavMeshObject.cpp

```diff
@@ -368,8 +368,12 @@ bool KX_NavMeshObject::BuildNavMesh()
 	const int dataSize = headerSize + vertsSize + polysSize + nodesSize +
 		detailMeshesSize + detailVertsSize + detailTrisSize;
 	unsigned char* data = new unsigned char[dataSize];
-	if (!data)
+	if (!data) {
```

Comment:
Well, this check seems useless finally as we don't check for every memory allocation, and that, i hope, the `dataSize`is never 0.